packages/liveblocks-react/src/comments/errors.ts

```diff
@@ -1,5 +1,8 @@
 import type { BaseMetadata, CommentBody, Patchable } from "@liveblocks/core";
 
+/**
+ * @private Internal API, do not rely on it.
+ */
```

Comment:
Is this really private? Can't it be used with `instanceof` and `useErrorListener`? (I'm not sure where we left off with error handling for Comments/Notifications)

Reply:
Maybe it's not intended to be private, but in that case, we should publish and document all of these:

https://github.com/liveblocks/liveblocks/blob/9029869472aba1e325584afb41ec6d86599fb2f4/packages/liveblocks-react/src/comments/errors.ts#L147-L154

Currently, only `CreateThreadError` is exported here, and given that it's in a separate re-export together with `useCommentsErrorListener`, which is also private, I figured it's at least supposed to be private _for now_.

Reply:
Ah yep, looks like it's a case of needing it in `react-lexical` again so `@private` indeed for now.